File: Generated_Codes_Check/Results/python/44.py
"""


"""

# cook your dish here
for i in range(int(input())):
    n,s=input().split()
    n=int(n)
    l=[]
    d={"CONTEST_WON":300,"TOP_CONTRIBUTOR":300,"BUG_FOUND":50,"CONTEST_HOSTED":50}
    for j in range(n):
        l.append(input().split())
    c=0
    for k in range(len(l)):
        if l[k][0]=="CONTEST_WON":
            c+=d["CONTEST_WON"]+(20-int(l[k][1]))
        else:
            c+=d[l[k][0]]
    if s=="INDIAN":
        print(int(c/200))
    else:
        print(int(c/400))

File: Scripts/generated_codes/java/44.java


import java.util.Scanner;

public class Main {
	public static void main(String[] args) {
		Scanner  input = new Scanner(System.in);
		int cases = input.nextInt();
		
			for(int i = 0; i < cases; i++) {
				int activities = input.nextInt();
				String origin = input.next();
				
				int laddus = 0;
				
				for(int j = 0; j < activities; j++) {
					String activity = input.next();
					if(activity.equals("CONTEST_WON")) {
						int rank = input.nextInt();
						laddus += 300;
						if(rank <= 20)
							laddus += 20-rank;
					}
					else if(activity.equals("TOP_CONTRIBUTOR"))
						laddus += 300;
					else if(activity.equals("BUG_FOUND")) {
						int severity = input.nextInt();
						laddus += severity;
					}
					else if(activity.equals("CONTEST_HOSTED"))
						laddus += 50;
				}
				
				int amount = 0;
				if(origin.equals("INDIAN"))
					amount = 200;
				else
					amount = 400;
				
				System.out.println(laddus/amount);
				
			}
	}
}